### DISCUSSION TITLE
Add Option to end user switch between eggs with same configuration.

### DISCUSSION THREAD

**Discussion by PedroPinMag** [pterodactyl/panel#4111] [Feature Requests]
Hello from Brazil 🇧🇷🇧🇷. I use pterodactyl for sharing the same machine between servers from different services with some friends. But the more friends came the more requests they're requesting.

The Minecraft 1.18 forge needs a .sh running capable egg but the paper 1.18 needs a .jar running capable egg, so whenever a user needs to change from those versions, they need to ask me (admin) to change the egg. It would be nice to add a feature that give end users the option to change the egg but keeping the build config (ram, CPU, allocations, backup, databases, etc)

Thanks by the work. You're awesome.

**Comment by trenutoo** [comment]
The forge egg doesn't need or use any shell scripts. It uses unix_args for JPMS 1.17+ and bundled jar for older versions. As a temporary solution to your friends; create an egg variable and replace the startup parameters with the variable, allowing them to modify the startup.

Also, there is an open suggestion for this https://github.com/pterodactyl/panel/issues/1257
